scripts/test_e2e.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Copyright 2021 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+set -o errexit
+set -o pipefail
+
+echo '''
```

Comment:
Shouldn't this be done in the `test-e2e` make task? Otherwise we run the risk of someone running `make test-e2e`, and them skipping this warning. No?

Reply:
I see your point. Was checking out other repos - calling a make task from the script seems to be the pattern

https://github.com/kubernetes-sigs/cluster-api/blob/18c7ec867f7e8eb690aa7708afe6bbc27fec1498/scripts/ci-e2e.sh#L100

https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/f7c3a7495d26e96fb50d83ff0d1994016e3b211e/hack/e2e.sh#L110

https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/2969152895a268fdaf022156e3b14777d9bcfd8f/scripts/ci-e2e.sh#L84

How about documenting how to run our e2e tests? No? People won't read you say?
Or call the script from the make task and execute ginkgo from the script? We could do it - and I don't have strong opinions - just doesn't look like what everybody else is doing. But neither do they have tests modifying the state. So. Your call.
@jamiemonserrate

Reply:
I am more aligned towards having a single way to run e2e tests. Not sure though, but it might be possible to do the same thing in `Makefile`. I just tried below to read the input
```
test-e2e: 
	@read -p "Do you want to proceed [Y/n]?" REPLY; \
	echo "Your reply is ", $$REPLY
```

Reply:
Okay then, warnings in the make target it is!
I'll update the PR.
Thank you!